apps/tasktracker/providers/pivotal.go

```diff
@@ -264,6 +264,9 @@ func (c *PivotalConnection) GetSprint(sprintID string) *serializers.Sprint {
 
 // GetSprintTaskList ...
 func (c *PivotalConnection) GetSprintTaskList(sprint serializers.Sprint) []serializers.Task {
+	if len(sprint.ID) < 1 {
```

Comment:
Please add a comment here!! Also, why are we checking the length of the sprint ID?

```suggestion
	if sprint.ID == "" {
```
Why not this?

Reply:
Done.